README.md

```diff
@@ -40,3 +40,4 @@ Feel free to submit pull requests or open issues to add any companies.
 | Twisted Rope | Buffalo | Agency Work | Javascript, PHP | http://twisted-rope.com/ |
 | Utilant | Buffalo | Property Insurance | Primarily C#.NET | http://public.utilant.com/ |
 | Value Centric | Orchard Park | Data Management | Primarily ColdFusion | http://www.valuecentric.com/ |
+| Zappos | Buffalo (DIG) | e-commerce | AWS, Java, Elixir, React | https://buffalo.zappos.biz |
```

Comment:
Doesn't seem to have an "Apply Here" link nor does the Zappos.com site list positions in Buffalo. Am I missing something ?

Reply:
Click "Learn More" to get to their posting page. Presumably they just don't have open positions currently.